### PR TITLE
Add search funcionality

### PR DESCRIPTION
**Pull request by Joao17Duarte**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -6,18 +7,29 @@ import SearchInput from './components/SearchInput/SearchInput'
...
     <div>
       <Header name="Movie Picker" />
 
-      <SearchInput labelText="Choose your Movie:" placeholder="Movie Name" />
+      <SearchInput
+        labelText="Choose your Movie:"
+        placeholder="Movie Name"
+        searchInputValue={searchInputValue}
+        setSearchInputValue={setSearchInputValue}
+      />
```
Comment (AnastasiyaVasileuskaya) [inline src/App.js:24]: Es fehlt hier noch name Attribute.

File: src/components/SearchInput/SearchInput.js [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
-      <input name={name} placeholder={placeholder} type="text" />
+      <input
+        value={searchInputValue}
+        placeholder={placeholder}
+        type="text"
+        onChange={event => setSearchInputValue(event.target.value)}
+      />
     </LabelGrid>
   )
 }
```
Comment (AnastasiyaVasileuskaya) [inline src/components/SearchInput/SearchInput.js:20]: Man könnte noch für diese Komponente Tests und Story schreiben.

File: src/App.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import { useState } from 'react'
 import './App.css'
```
Comment (iammnils) [inline src/App.js:2]: Due to the fact that you are working with styled-components, you will most probably not make use of the App.css and its import.

File: src/App.js [hunk 4]
```diff
@@ -6,18 +7,29 @@ import SearchInput from './components/SearchInput/SearchInput'
 import data from './movieList.json'
 
 function App() {
+  const [searchInputValue, setSearchInputValue] = useState('')
+
   const tag = []
   data.map(({ genre }) => genre.map(item => tag.push(item)))
   const uniqueTag = [...new Set(tag)]
```
Comment (iammnils) [inline src/App.js:14]: Cool solution

**Review by iammnils — COMMENTED** [review]
Solid feature, well done! 👾 Just one comment left regarding your css files

**Pull request merged**